group-coordinator/src/main/java/org/apache/kafka/coordinator/group/modern/consumer/ConsumerGroup.java

```diff
@@ -59,11 +61,14 @@
 
 import static org.apache.kafka.coordinator.group.Utils.toOptional;
 import static org.apache.kafka.coordinator.group.Utils.toTopicPartitionMap;
+import static org.apache.kafka.coordinator.group.api.assignor.SubscriptionType.HETEROGENEOUS;
```

Comment:
btw, when will we remove `@InterfaceStability.Unstable` from the `SubscriptionType` class?

Reply:
I will remove them just before the code freeze.

Reply:
https://issues.apache.org/jira/browse/KAFKA-18169